posthog/src/main/java/com/posthog/PostHog.kt

```diff
@@ -227,6 +231,24 @@ public class PostHog private constructor(
             getPreferences().setValue(DISTINCT_ID, value)
         }
 
+    private var isIdentified: Boolean = false
+        @JvmName("get-isIdentified")
+        get() {
+            synchronized(identifiedLock) {
+                if (!isIdentifiedLoaded) {
+                    println("hi")
+                    this.isIdentified = getPreferences().getValue(IS_IDENTIFIED) as? Boolean
+                        ?: (this.distinctId != this.anonymousId)
+                    isIdentifiedLoaded = true
+                }
+            }
+            return field
+        }
+        set(value) {
+            field = value
+            getPreferences().setValue(IS_IDENTIFIED, value)
```

Comment:
I guess we should sync this with `identifiedLock` as well

Reply:
okay 👍